### ISSUE TITLE
6 voice polyphony only...

### ISSUE THREAD

**Issue by Airlane1979** [TheWaveWarden/odin2#434]
Using Odin 2 V 2.3.3 in Windows 10, I can get only 6 voices of polyphony in Unison 1 mode, single oscillator. Is this a bug?

**Comment by TheWaveWarden** [comment]
Yeah, it should be 24 voices. Can you provide some more info, maybe a screencast or something?

**Comment by TheWaveWarden** [comment]
Good to hear, thanks for reporting back!
